### DISCUSSION TITLE
How to give parameters to node startup

### DISCUSSION THREAD

**Discussion by dyn-do** [vercel/community#1199] [Help] [answered]
### Summary

I am creating a website with sveltekit.
I am also using three.js.
However, when I published my site with Vercel, I got the error importing three.js discussed below.
https://discourse.threejs.org/t/gltfloader-cannot-be-found/42254
No error occurred when I built it on my PC and started node in the following way.
`node --es-module-specifier-resolution=node build/index.js`
So I want to start node with `--es-module-specifier-resolution` parameter even in Vercel.
How can I set the startup parameters for node?

**Comment by swarnava** [comment, marked as answer]
Hi,

You can just add a script to your package.json:
Assuming index.js is your entry point and package.json is in the same directory

```json
{
  "scripts": {
    "start": "node --experimental-specifier-resolution=node build/index.js"
  }
}
```

Then you can just use `start` comand

**Reply by dyn-do** [reply]
Sorry for the late reply.
I got rid of the error by following your instructions.
Thank you very much.
